http-clients/src/main/java/org/triplea/http/client/github/issues/GithubIssueClient.java

```diff
@@ -33,6 +33,13 @@ public GithubIssueClient(
     this.authToken = authToken;
   }
 
+  /**
+   * For local or integration testing, we may want to have a fake that does not actually
+   * call github. This method returns true if we are doing a fake call to github.
+   */
+  public boolean isTest() {
+    return authToken.equalsIgnoreCase("test");
```

Comment:
Here is where we detect a magic value, kinda dirty, but it does allow for us to have an out-of-the-box configuration that is working and can be used for local test. The production application configuration does not default the environment variable to 'test', and we have an 'prod' check that will doubly make sure we do not use a magic value that triggers stubbing prod.

Reply:
The isTest method is later used to detect if we should use stub behavior, or if we should use github webservices API to create an issue.